frontend/src/pages/index.vue

```diff
@@ -89,6 +89,15 @@ function versions(platform: Platform) {
   });
 }
 
+function updatePlatform(platform: any) {
```

Comment:
Pretty sure the only line in that method would have to be `filters.value.versions = []`?

Reply:
As stated in the commit, I chose to keep versions that technically can be re-used specifically for switching between paper and waterfall. As we share the same version with waterfall, resetting it all the time felt off to me.

Reply:
Ah right. Yeah that's okay